docs/reference/mapping/types/runtime.asciidoc

```diff
@@ -0,0 +1,90 @@
+[[runtime]]
+=== Runtime
+
+////
+[source,console]
+----
+PUT /my-index
+----
+// TESTSETUP
+////
+
+Typically, you must index fields to {es} before they can be retrieved,
+aggregated, or searched. With runtime fields, you can explicitly define a field
+in the mapping and access it at runtime without indexing your data. This
+flexibility allows you to more quickly ingest raw data into the Elastic Stack
+and immediately access it. By dynamically evaluating runtime fields at query
+time, you can choose which fields to index and optimize disk space.
+
+Runtime fields are not indexed, but are accessible from the search API like any
+other field that has <<doc-values,doc_values>> and is searchable. You can
+retrieve and query these fields, as well as aggregate and
+sort on them.
+
+NOTE: Runtime fields can be incur performance costs at search time, depending
+on the <<runtime-params-runtime-type,`runtime_type`>>.
+
+==== Specifying runtime fields
+You specify runtime fields in the mapping by
```

Comment:
We typically call this "mapping a field." "Specifying" doesn't feel right.

Reply:
Thanks for clarifying. I'll update the wording.